docker/mbzirc_sim/Dockerfile

```diff
@@ -99,12 +105,6 @@ RUN mkdir ~/workspaces ~/other
 # When running a container start in the developer's home folder
 WORKDIR /home/$USERNAME
 
-# clone mbzirc sim
-RUN mkdir -p mbzirc_ws/src \
- && cd mbzirc_ws/src \
- && git config --global http.postBuffer 1048576000 \
- && git clone https://github.com/osrf/mbzirc
```

Comment:
we shouldn't remove this in this PR right? Or maybe #18 should be target at this branch

Reply:
Sorry the intention was to open two PRs against main but I failed to do it properly. Let's do this and I reopen the next one.

Reply:
ok I reverted this line otherwise the next cmd for cd'ing into the directory will fail. a8203c3